src/app/user/login/post.controller.js

```diff
@@ -76,7 +76,7 @@ module.exports = (req, res) => {
       });
     })
     .catch((err) => {
-      logger.error('Validation error when logging in');
+      logger.info('Validation error when logging in');
```

Comment:
Validation errors are not errors in terms of exceptions etc, so normally they are logged at INFO level.